forms-web-app/tests/unit/controllers/appellant-submission/site-access-safety.test.js

```diff
@@ -92,35 +105,79 @@ describe('controllers/appellant-submission/site-access-safety', () => {
 
       expect(res.redirect).toHaveBeenCalledWith(`/${VIEW.APPELLANT_SUBMISSION.TASK_LIST}`);
 
-      const { empty: appeal } = APPEAL_DOCUMENT;
-      appeal[sectionName][taskName].hasIssues = true;
```

Comment:
it turns out you can change any of these values and the tests still pass :exploding_head: